docs/Changelog.md

```diff
@@ -2,6 +2,21 @@
 
 <!-- See rendered changelog at https://streamly.composewell.com -->
 
+## 0.8.2 (Feb 2022)
+
```

Comment:
We can mention the performance fixes for GHC 9.2. The performance will be on par with previous compilers once GHC fixes land in GHC 9.2.2.